docs/usage/post_training_compression/weights_compression/Usage.md

```diff
@@ -527,6 +527,251 @@ Here is the perplexity and accuracy with data-free and data-aware mixed-precisio
     <t/r>
 </table>
 
+Below are the tables showing the accuracy/footprint trade-off for 2 models `Qwen/Qwen2-7B` and
+`microsoft/Phi-3-mini-4k-instruct` compressed with different options.
+
+Footprint overhead is measured by the percentage increase in model size relative to a baseline int4 model, compressed with `ratio=1` and `all_layers=False` options.
+Accuracy metrics are measured on 4 tasks [lambada openai](https://huggingface.co/datasets/EleutherAI/lambada_openai), [wikitext](https://arxiv.org/pdf/1609.07843.pdf),
+[winogrande](https://arxiv.org/abs/1907.10641), [WWB](https://github.com/openvinotoolkit/openvino.genai/tree/master/llm_bench/python/who_what_benchmark/whowhatbench).
+The `average relative error` in the tables below is the mean of relative errors for each of four tasks with respect to
+the metric value for fp32 model. All int4 models are compressed group-wise with `group_size=128` and `mode=CompressionMode.INT4_SYM` and
+with calibration dataset based on 128 samples from `wikitext-2-v1`. Int8 model is compressed with `mode=CompressionMode.INT8_ASYM`.
+The following advanced parameters were used for AWQ, Scale Estimation and Lora Correction algorithms:
+
+```python
+AdvancedCompressionParameters(
+  awq_params=AdvancedAWQParameters(32, 0.05, 0.0, 1.0, 100),
+  scale_estimation_params=AdvancedScaleEstimationParameters(32, 5, 10, -1.0),
+  lora_correction_params=AdvancedLoraCorrectionParameters(adapter_rank=<LORA_RANK>)
+)
+```
+
+The tables clearly shows the followings:
+
+- More layers in 8 bit does improve accuracy, but it increases the footprint a lot.
+- Scale Estimation, AWQ, GPTQ do improve accuracy of the baseline int4 model without footprint increase.
+- Lora correction algorithm improves the accuracy of int4 models further with a footprint much less compared to mixed-precision models with the same or worse accuracy.
+
+Accuracy/footprint trade-off for `Qwen/Qwen2-7B`:
+
+<div class="tg-wrap"><table><thead>
+  <tr>
+    <th>Mode </th>
+    <th>%int4</th>
+    <th>%int8</th>
+    <th>lora <br>     rank</th>
+    <th>avg<br>relative<br>     error<br></th>
+    <th>model size<br>     overhead</th>
```

Comment:
Am I right that adding even the relative latency overhead in percents is not possible without specific approval? 
@MaximProshin @AlexKoff88